tabular/src/autogluon/tabular/models/tab_transformer/hyperparameters/parameters.py

```diff
@@ -0,0 +1,58 @@
+from ....constants import BINARY, MULTICLASS, REGRESSION
+
+
+def get_fixed_params():
+    """ Parameters that currently cannot be searched during HPO
+    TODO: HPO NOT CURRENTLY IMPLEMENTED FOR TABTRANSFORMER
+    Will need to figure out what (in future PR) is "fixed" and what is searchable. """
+    fixed_params = {'batch_size': 512,
+                    'tab_kwargs': {'n_cont_embeddings': 0,
```

Comment:
general comment: how do we expect people to update values here? If they just want to change hidden_dim to 96, wouldn't they have to also specify all the other values too?

Reply:
This is good feedback, I'll un-pack the params in tab_kwargs.